app/javascript/components/CreateWork.jsx

```diff
@@ -102,31 +156,35 @@ class CreateWork extends React.Component {
                 required
               />
             </p>
-            <p className="pt-ui-text">Medium:
-              <input
-                value={this.state.work.medium}
-                onChange={this.handleChange}
-                name="medium"
-                type="text"
-                className="pt-input"
-                required
-              />
-            </p>
-            <p className="pt-ui-text">Availability:
-              <input
-                value={this.state.work.availability}
-                onChange={this.handleChange}
-                name="availability"
-                type="text"
-                className="pt-input"
-                required
-              />
-            </p>
+            <div className="drop-down"> Medium:
+                <select onChange={this.mediumHandleChange} value={this.state.work.medium}>{
+                   Object.keys(this.state.categories.medium).map((obj) => {
+                       return <option>{obj}</option>
+                   })
+                }</select>
+              </div>
+              <div className="drop-down"> Availability:
+                <select onChange={this.availabilityHandleChange} value={this.state.work.availability}>{
+                   Object.keys(this.state.categories.availability).map((obj) => {
+                       return <option>{obj}</option>
+                   })
+                }</select>
+              </div>
+
             <div className="upload-image-component">Images:
               <Dropzone
                 onDrop={this.onDrop}
                 multiple={true}
               />
+              <div>
+                {this.state.work.images.map((img) => {
+                  return (
+                    <div>
+                      <div>{img.name}</div>
+                      <button onClick={() => {this.selectFeaturedImage(img)}}>Select as Featured Image</button>
```

Comment:
This only appears within the create work form, when it should appear in both. but i think this should be fixed once you make it a new component

Reply:
fixed!